src/components/global/Footer.jsx

```diff
@@ -130,73 +149,48 @@ const logoBlobStyle = {
   left: 0,
   right: 0,
   top: 0,
+  backgroundSize: 'cover',
   backgroundImage: { md: 'url(/assets/logoBlob.svg)', sm: 'none' },
   backgroundRepeat: 'no-repeat',
-  backgroundPosition: 'left bottom',
   zIndex: '-1',
   display: {
-    lg: 'block',
+    lg: 'flex',
     xs: 'none'
-  }
+  },
+  maxWidth: '200px'
 };
 
 const Footer = () => {
   const theme = useTheme();
 
   return (
-    // this box contains the entire footer
-    <Box
-      component="footer"
-      sx={{
-        display: 'flex',
-        flexDirection: { xs: 'column', sm: 'row' },
-        justifyContent: 'space-between',
-        alignItems: 'center',
-        position: 'fixed',
-        bottom: 0,
-        width: '100vw',
-        height: { xs: 'auto', sm: '200px' }
-      }}
-    >
+    // this container contains the entire footer
+    <Container component="footer" maxWidth="100%" sx={footerContainerStyle}>
+      {/* this box contains the blob behind the roseLogo */}
       <Box sx={logoBlobStyle}></Box>
-      <Stack direction={{ xs: 'row', sm: 'column', lg: 'row' }}>
+      <Stack direction={{ xs: 'row', sm: 'column', lg: 'row' }} flex={1}>
         {/* roseLogo box */}
         <Box
           component="img"
           alt="CODE PDX logo"
+          aria-label="Code pdx"
           src={'/assets/rose_logo.png'}
           sx={{
             width: '75px',
             mt: '25px',
-            ml: { xs: '0px', sm: '40px', lg: '50px' }
+            ml: { xs: '0px', sm: '40px', lg: '35px' }
           }}
         />
-        <Typography ml={{ xs: '10px', lg: '60px' }} mt="35px" variant="h5">
+        <Typography ml={{ xs: '10px', lg: '80px' }} mt="35px" variant="h5">
           CODE PDX
         </Typography>
       </Stack>
-
-      {/* this box and stack contain the social icons and blob*/}
-      <Box
-        sx={{
-          position: 'absolute', // Absolute positioning
-          left: '50%', // Centering
-          top: '50%',
-          transform: 'translate(-50%, -50%)',
-          display: 'flex',
-          minWidth: '1000px',
-          zIndex: -1,
-          [theme.breakpoints.down('sm')]: {
-            position: 'static', // Remove absolute positioning
-            transform: 'none' // Reset transform
-          }
-        }}
-      >
-        <Box sx={socialBlobStyle}>{renderSocialLinks}</Box>
-      </Box>
-      {/* this stack contains the legal links and hook to add padding on viewport scale down */}
+      {/* this box contains social links*/}
+      <Box display={'flex'}>{renderSocialLinks}</Box>
```

Comment:
Make this instead a `<Stack direction="row" spacing={{xs: 2, sm: 3}}>{renderSocialLinks}</Stack>`.     You could play with the spacing on `sm` if you want it bigger, but I didn't test any bigger than this.   `Stack` is flex by default.